### PR TITLE
Add SE view

### PR DESCRIPTION
**Pull request by DadImScared**

### CODE REVIEW

**Comment by DadImScared** [comment]
[SE: Tasks listing](https://trello.com/c/6ZZ2axlG/24-se-tasks-listing)

File: frontend/src/components/SQRDataTable.vue [hunk 1]
```diff
@@ -28,13 +39,15 @@
 <script lang="ts">
 import { Component, Vue, Prop } from "vue-property-decorator";
 import _ from "lodash";
+import SQRTableData from "@/components/SQRTableData";
```
Comment (anantakrishna) [inline frontend/src/components/SQRDataTable.vue]: As soon as this component is not SQR-specific, consider naming it in a more generic way.

File: frontend/src/main.ts [hunk 2]
```diff
@@ -26,7 +26,8 @@ Vue.use(Vuetify, {
   icons: {
     listening: "fas fa-headphones",
     track: "fas fa-cut",
-    quality: "fas fa-check"
+    quality: "fas fa-check",
+    musicNotes: "fas fa-music"
```
Comment (anantakrishna) [inline frontend/src/main.ts]: The name of the custom icon should be semantic. See the previous ones. I propose renaming this to `sound`.

File: frontend/src/components/TableData.ts [hunk 3]
```diff
@@ -1,9 +1,9 @@
 import { Component, Prop, Vue } from "vue-property-decorator";
 
 @Component({
-  name: "SQRTableData"
+  name: "TableData"
 })
-export default class SQRTableData extends Vue {
+export default class TableData extends Vue {
```
Comment (anantakrishna) [inline frontend/src/components/TableData.ts]: Just curious: why `TableData` and not `DataTable`?

Comment (DadImScared) [reply frontend/src/components/TableData.ts]: DataTable.vue wraps v-data-table and TableData is used inside the `<td>` tag

**Pull request merged**